packages/web3-eth/test/integration/watch_transaction.test.ts

```diff
@@ -85,7 +80,7 @@ describeIf(isWs)('watch subscription transaction', () => {
 				// Tx promise is handled separately
 				// eslint-disable-next-line no-void
 				void sentTx.on('confirmation', ({ confirmations }) => {
-					expect(Number(confirmations)).toBe(shouldBe);
+					expect(Number(confirmations)).toBeGreaterThanOrEqual(shouldBe);
```

Comment:
Why it should be here greater than or equal? Was not the logic here is to check if the confirmations arrive in order? (1 then 2 then 3...) If we will check just the greater-than-or-equal, we do not need other than this: right?
```
if (confirmations >= waitConfirmations) {
    resolve();
}
```

(This applies also to the changes in the next file.)

Reply:
When I run tests in parallel, it mined a lot of transactions in other tests at the same time. so I receive a lot of confirmations at the  same time